provider/execution/execution.go

```diff
@@ -87,7 +87,8 @@ func (e TableExecutor) Resolve(ctx context.Context, meta schema.ClientMeta) (uin
 		ctx, cancel = context.WithTimeout(ctx, e.timeout)
 		defer cancel()
 	}
-	return e.callTableResolve(ctx, meta, nil)
+	logger := meta.Logger().With("table", e.Table.Name, "multiplex", false)
+	return e.callTableResolve(ctx, logger, meta, nil)
```

Comment:
would actually do e.WithLogger(...).callTableResolver and inside execution call the logger instead, as you want the table to also change when going to sub relation or do  resource: "X" and table: "Y" where Y can change to relation.

Moreover would also add to the call the multiplex info i.e in aws case region + account_id as you can have multiple execution of a same resource but you want to distinguish them by region/account_id

Reply:
Same comment applies here. This is the best we can do without breaking interfaces.

Reply:
I think a with would be cleaner then passing logger to all methods. I would actually break yes, add to client.Meta a new interface method called Identifiers() []Identifiers-> [account_id, region], otherwise we wont be able to consistently distinguish specific breaking clients by their identifier / problematic i.e it can be a specific account / region doing issues etc'

Reply:
with wouldn't work with all methods as it would create new execution?